README.md

```diff
@@ -1,79 +1,142 @@
 
-## Sample Screenshots of Runtime
+However, if you checked the first box above, you are running Kubernetes on-premise and will need to manage your own load balancer. This is where NKL comes in.
 
-<br/>
+NKL itself does not perform load balancing. Instead, NKL allows you to manage resources within your cluster and have the load balancers automatically be updated to support those changes, with tooling you are most likely already using. 
 
-### Configuration with 2 Nginx LB Servers defined (HA):
+## Getting Started
 
-![NGINX LB ConfigMap](docs/media/nkl-pod-configmap.png)
+There are few bits of administrivia to get out of the way before you can start leveraging NKL for your load balancing needs.
 
-<br/>
+As noted above, NKL really shines when you have one or more Kubernetes clusters running on-premise. With this in place,
+you need to have at least one NGINX Plus host running outside your cluster (Please refer to the [Roadmap](#Roadmap) for information about other load balancer servers). 
 
-### Nginx LB Server Dashboard and Logging
+You will not need to clone this repo to use NKL. Instead, you can install NKL using the included Manifest files (just copy the `deployments/` directory), which pulls the NKL image from the Container Registry.
 
-![NGINX LB Create Nodeport](docs/media/nkl-create-nodeport.png)
+### RBAC
 
-Legend:
-- Red - kubectl commands
-- Blue - nodeport and upstreams for http traffic
-- Indigo - nodeport and upstreams for https traffic
-- Green - logs for api calls to LB Server #1
-- Orange - Nginx LB Server upstream dashboard details
-- Kubernetes Worker Nodes are 10.1.1.8 and 10.1.1.10
+As with everything Kubernetes, NKL requires RBAC permissions to function properly. The necessary resources are defined in the various YAML files in `deployement/rbac/`.
 
-<br/>
+For convenience, two scripts are included, `apply.sh`, and `unapply.sh`. These scripts will apply or remove the RBAC resources, respectively.
 
-## Requirements
+The permissions required by NKL are modest. NKL requires the ability to read Resources via shared informers; the resources are Services, Nodes, and ConfigMaps. 
+The Services and ConfigMap are restricted to a specific namespace (default: "nkl"). The Nodes resource is cluster-wide.
+
+### Configuration
+
+NKL is configured via a ConfigMap, the default settings are found in `deployment/configmap.yaml`. Presently there is a single configuration value exposed in the ConfigMap, `nginx-hosts`. 
+This contains a comma-separated list of NGINX Plus hosts that NKL will maintain.
+
+You will need to update this ConfigMap to reflect the NGINX Plus hosts you wish to manage.
+
+If you were to deploy the ConfigMap and start NKL without updating the `nginx-hosts` value, don't fear; the ConfigMap resource is monitored for changes and NKL will update the NGINX Plus hosts accordingly when the resource is changed, no restart required.
+
+### Deployment
```

Comment:
I'd suggest weaving in the rbac and configuration parts to the step by step here rather than putting them up front.  They seemed to dangle a bit